src/routes/Profile/WorkHistory/WorkHistoryWizard/WorkHistoryWizard.tsx

```diff
@@ -16,11 +14,10 @@ import { type WorkHistoryForm } from './utils';
 
 interface WorkHistoryWizardProps {
   isOpen: boolean;
-  setIsOpen: React.Dispatch<React.SetStateAction<boolean>>;
-  selectedRow?: SelectableTableRow;
+  setIsOpen: () => void;
```

Comment:
Is this now more like `onClose()`? The current `setIsOpen` name is somewhat misleading with the boolean in it's name and set, when it takes no parameters.

Reply:
Renamed these ones too